spec/quickbooks-payments/request_spec.rb

```diff
@@ -37,7 +41,7 @@
         it 'sends args to access_token' do
           Quickbooks::Payments.access_token = access_token
           expect(Quickbooks::Payments.access_token).to receive(meth)
-            .with(args[0], '', default_headers)
+            .with(get_url(args[0]), '', default_headers)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.